tests/webtest/test_CopyAsMarkdown.js

```diff
@@ -0,0 +1,246 @@
+  TestUtils.keypress(frame.contentDocument.documentElement, { bubbles: true, key: "S" });
+  is(copiedText, "*unmodified*", "Copy does not happen for disabled Symbol Link accel");
+
+  // Select the global variable.
+  selectLine(5);
+
+  ok(!filenameButton.disabled, "Filename Link should always be enabled");
+  ok(!symbolButton.disabled, "Symbol Link should be enabled when a symbol exists in the selected line");
+  ok(!codeButton.disabled, "Code Block should be enabled when a line is selected");
+
+  TestUtils.click(symbolButton);
+  is(sanitizeURL(copiedText),
+     "[globalVariable](BASE_URL/tests/source/webtest/CopyAsMarkdown.cpp#5)",
+     "Global variable symbol is copied");
+
+  copiedText = "";
+  TestUtils.keypress(frame.contentDocument.documentElement, { bubbles: true, key: "S" });
+  is(sanitizeURL(copiedText),
+     "[globalVariable](BASE_URL/tests/source/webtest/CopyAsMarkdown.cpp#5)",
+     "Global variable symbol is copied");
+
+  // Select the namespace.
+
+  selectLine(7);
+
+  ok(!filenameButton.disabled, "Filename Link should always be enabled");
+  ok(!symbolButton.disabled, "Symbol Link should be enabled when the selected line is inside a nesting line");
+  ok(!codeButton.disabled, "Code Block should be enabled when a line is selected");
+
+  TestUtils.click(symbolButton);
+  is(sanitizeURL(copiedText),
+     "[copy_as_markdown](BASE_URL/tests/source/webtest/CopyAsMarkdown.cpp#7)",
+     "Namespace symbol is copied");
+
+  // Select the comment inside namespace.
+  selectLine(9);
+
+  ok(!filenameButton.disabled, "Filename Link should always be enabled");
+  ok(!symbolButton.disabled, "Symbol Link should be enabled when the selected line is inside a nesting line");
+  ok(!codeButton.disabled, "Code Block should be enabled when a line is selected");
+
+  // Select the method
+  selectLine(15);
+
+  ok(!filenameButton.disabled, "Filename Link should always be enabled");
+  ok(!symbolButton.disabled, "Symbol Link should be enabled when the selected line is inside a nesting line");
+  ok(!codeButton.disabled, "Code Block should be enabled when a line is selected");
+
+  TestUtils.click(symbolButton);
+  is(sanitizeURL(copiedText),
+     "[copy_as_markdown::CopyAsMarkdown::SomeMethod](BASE_URL/tests/source/webtest/CopyAsMarkdown.cpp#15)",
+     "Method symbol is copied");
+
+  // Select the local variable
+  selectLine(19);
+
+  ok(!filenameButton.disabled, "Filename Link should always be enabled");
+  ok(!symbolButton.disabled, "Symbol Link should be enabled when the selected line is inside a nesting line");
+  ok(!codeButton.disabled, "Code Block should be enabled when a line is selected");
+
+  TestUtils.click(symbolButton);
+  is(sanitizeURL(copiedText),
+     "[copy_as_markdown::CopyAsMarkdown::SomeMethod](BASE_URL/tests/source/webtest/CopyAsMarkdown.cpp#19)",
+     "Method symbol is copied instead of local variable symbol");
+
+  // Select the local variable
```

Comment:
Nit: Maybe this comment could be something like:

Shift-select from the start of the class to the currently selected local variable.